src/docs/sdk/performance/opentelemetry.mdx

```diff
@@ -18,7 +18,220 @@ When Sentry performance monitoring was initially introduced, OpenTelemetry was i
+import { SentrySpanProcessor } from "@sentry/opentelemetry-node";
+
+Sentry.init({
+  /// ...
+});
+
+const sdk = new NodeSDK({
+  resource: new Resource({
+    "service.name": "my-service",
+    "service.version": "1.0.0",
+  }),
+  spanProcessor: new SentrySpanProcessor(),
+});
+```
+
+### Step 2: Define `getTraceData`
+
+We want to make sure a transaction is started with information from the `sentry-trace` and `baggage` headers.
+
+TODO @neel
+
+### Step 3: Define `updateSpanWithOtelData`
+
+We want to update the Sentry Span with the OpenTelemetry data. This includes the span's operation, description, and resource attributes.
+
+The Sentry span description should come from the OpenTelemetry Span name. The Sentry span operation should come from a combinations of the OpenTelemetry Span name, kind, and attributes.
+
+To make things simple, only set ops for `db` and `http` spans. Don't do any other extended logic, this will be done in Relay in the future.
+
+### Step 3: Add OpenTelemetry Context
+
+All SDKs are required to add event context with the key `otel` to events generated by the SpanProcessor. This is detailed in <Link to="./#opentelemetry-context">our spec for the Context</Link>. The OpenTelemetryContext should be generated from information from the OpenTelemetry Span.
+
+```ts
+{
+  contexts: {
+    otel: {
+      attributes: {
+        "http.method": "GET",
+        "http.url": "https://example.com",
+        "http.status_code": 200,
+      },
+      service: {
+        name: "my-service",
+        version: "1.0.0",
+      },
+      otel_sdk: {
+        name: "opentelemetry-node"
+        version: "4.0.0",
+      },
+      // ...
+    }
+  },
+}
+```
+
+### Step 4: Add instrumentor option to Sentry SDK
+
+We want to avoid double instrumenting the same library. To do this, we want to add an option to the Sentry SDK that disables Sentry performance auto-instrumentation when OpenTelemetry is enabled. For now users will have to manually add this option to their SDK initialization code, but in the future we can do this automatically (by detecting if OpenTelemetry exists).
+
+```ts
+Sentry.init({
+  // ...
+  instrumentor: "otel",
+});
+```
+
+You have two options for this:
+
+1. Add an early return to `Sentry.startTransaction` and `Sentry.trace` (a helper method that ) if the `instrumentor` option is set to `otel`. This may require SDK changes, as `trace` is not part of the unified API.
+
+```ts
+class Hub implements HubInterface {
+  // ...
+  startTransaction(
+    this: Hub,
+    context: TransactionContext
+  ): Transaction | undefined {
+    // ...
+    if (this.instrumentor !== context.instrumentor) {
+      return;
+    }
+    // ...
+  }
+
+  // Note this is not part of the unified API.
+  // It is being proposed as part of:
+  // https://github.com/getsentry/rfcs/issues/28
+  // see: https://github.com/getsentry/develop/issues/720
+  trace<T>(this: Hub, context: SpanContext, callback: (span?: Span) => T): T {
```

Comment:
I am not sure if we should mention `Sentry.trace()` here, because it is not important for the first version of the Otel support (Because Otel support should work for projects that do not use `Sentry.trace()`. (Or should we internally use `Sentry.trace()` in the new release of the SDKs that support otel?

Reply:
I don't think having `Sentry.trace` blocks you from the first version of an OTEL SDK since you can do Option 2 (add conditionals in callsites).

Ideally though you would adopt `Sentry.trace`, but we have to RFC this and gather more feedback. We should have a better idea about this by end of week.

Reply:
for my understanding: tracing is an important if not equally important to spans, I understand traces without spans doesn't make much sense, but regardless.

Is the discussion here around if we should just focus on span transformation and leave tracing for later? Or am i missing the point?

Reply:
The discussion here is re: https://github.com/getsentry/develop/issues/720